whatsapp.py

```diff
@@ -122,6 +122,27 @@ def count_curses(df):
     counter = counter.transpose()
     return counter
 
+def counter_by_user(df,media_df):
+    ''' all the counters in one df
+        currently: messages, words, hhh, emoji, questions
+        all by authors
+        '''
+    counter = pd.DataFrame()
+    counter["messages"] = df.groupby("author").count().iloc[:,1]
+    counter["media"] = media_df.groupby("author").count().iloc[:,1]
+    counter["words"] = None
+    for author in Message.authors:
+        authors_words = " ".join(df[df['author'] == author]["text"]).split()  # Filter here if you want to remove "yes" and whatever
```

Comment:
I don't know if there's a way to open an issue from a comment, but here we want to add another way to normalize based on frequency in langugage

Reply:
Don't know either. want to open the issue in issues? I can to, but you have yet to open one.

Reply:
opened issue